modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.cc

```diff
@@ -442,39 +432,38 @@ LBSSolver::OptionsBlock()
 
   params.SetGeneralDescription("Set options from a large list of parameters");
   params.SetDocGroup("LBSUtilities");
-
   params.AddOptionalParameter("spatial_discretization",
                               "pwld",
                               "What spatial discretization to use. Currently only `\"pwld\"` "
                               "is supported");
   params.AddOptionalParameter(
     "scattering_order", 1, "Defines the level of harmonic expansion for the scattering source.");
   params.AddOptionalParameter("max_mpi_message_size",
-                              32'768,
+                              32768,
                               "The maximum MPI message size used during sweep initialization.");
   params.AddOptionalParameter(
     "read_restart_data", false, "Flag indicating whether restart data is to be read.");
+  params.AddOptionalParameter("read_restart_directory_name",
+                              opensn::input_path.stem().string() + "_restart",
+                              "Directory name to use when reading restart data.");
+  params.AddOptionalParameter("read_restart_file_stem",
+                              opensn::input_path.stem().string(),
+                              "File stem to use when reading restart data.");
+  params.AddOptionalParameter("write_restart_directory_name",
+                              opensn::input_path.stem().string() + "_restart",
+                              "Directory name to use when writing restart data.");
   params.AddOptionalParameter(
-    "read_restart_folder_name", "YRestart", "Folder name to use when reading restart data.");
-  params.AddOptionalParameter(
-    "read_restart_file_base", "restart", "File base name to use when reading restart data.");
-  params.AddOptionalParameter(
-    "write_restart_data", false, "Flag indicating whether restart data is to be written.");
-  params.AddOptionalParameter(
-    "write_restart_folder_name", "YRestart", "Folder name to use when writing restart data.");
-  params.AddOptionalParameter(
-    "write_restart_file_base", "restart", "File base name to use when writing restart data.");
-  params.AddOptionalParameter(
-    "write_restart_interval",
-    30.0,
-    "Interval at which restart data is to be written. Currently not implemented.");
+    "write_restart_file_stem", "restart", "File stem to use when writing restart data.");
+  params.AddOptionalParameter("write_restart_time_interval",
+                              60,
+                              "Time interval in seconds at which restart data is to be written.");
+  params.ConstrainParameterRange("write_restart_time_interval", AllowableRangeLowLimit::New(60));
```

Comment:
```suggestion
  params.AddOptionalParameter("restart_dump_frequency",
                              60,
                              "Frequency in seconds to write restart dumps.");
  params.ConstrainParameterRange("write_restart_time_interval", AllowableRangeLowLimit::New(60));
```
Should we default to writing restart dumps at the smallest allowable interval?

Reply:
Good point. Maybe something like every 10 minutes?

Reply:
That depends... Are restarts always on? If so, 10 minutes seems okay. If not, then I would default to 0 (get rid of the minimum) and check that it is non-zero whenever restarts are enabled (however that is done).

Reply:
I defaulted to every 10 minutes. That seems like a reasonable value. Presumably restart dumps will be enabled for mostly long-running problems.